plugins/workspace-backpack/src/backpack_helpers.js

```diff
@@ -10,6 +10,171 @@
  */
 
 import * as Blockly from 'blockly/core';
+import './msg';
+
+/**
+ * Registers a context menu option to empty the backpack when right-clicked.
+ * @param {!Blockly.WorkspaceSvg} workspace The workspace to register the
+ *   context menu option on.
+ */
+function registerEmptyBackpack(workspace) {
```

Comment:
I've added the call to the previous context menu implementation if the backpack is not clicked. 

I added a section to the README on "Compatibility" to let the developer know that they need to call the old `workspace.configureContextMenu`. However, I agree that it would be better if we had a hook in core, like an additional `scopeType`.